decentraland/unity-renderer
# feat: add toast for avatar modifier

## What does this PR change?

Add a toast describing the player's state when entering an Avatar Modifier Area. 

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=chore/add-toast-for-avatar-modifier&position=-29,56. There is an Avatar Modifier Area in front of the Wondermine machine
2. Get into the area. Check that the messages fade and animates. After 5 seconds, it should fade out to the warning icon. This behavior happens only the first time a user enters an Avatar Modifier Area.
3. Check that the message says "The avatars are hidden"
4. When focusing on the icon, the message should reappear
5. When leaving the area, the icon should fade out
6. When re-entering the area, the icon should fade in. 

Message View
![image](https://user-images.githubusercontent.com/1999557/178157434-f4b595bc-cf98-43a1-8392-949ca5a8d919.png)


Icon View
![image](https://user-images.githubusercontent.com/1999557/178157420-473b241c-8b70-4188-ab53-8c2100ff68a7.png)


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
